plugin.video.filmsforaction/README.md

```diff
@@ -0,0 +1,46 @@
+Films For Action Videos: XBMC video add-on to watch the Films For Action videos from the FFA web site.
```

Comment:
Kodi :)

Reply:
You are right. The add-on is quite old and didn't change the name. I'll change it for the next PR.